### PR TITLE
Clearer Contributing.MD

### PR DESCRIPTION
**Pull request by talsraviv**
Will submit here first for discussion and then transfer over to Goodies

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by talsraviv** [comment]
Would like to add the WIP tag if possible - more on the way

**Comment by talsraviv** [comment]
@GuiltyDolphin @moollaza - ready to merge, just went over with @zekiel

**Comment by GuiltyDolphin** [comment]
@talsraviv Sweet, thanks! :dolphin:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,42 +1,59 @@
...
 
-	- "**Fixes #2038.** The images used by the API are very large and don't change often. I've but a smaller version of each image (and a 2x version for retina screens) in the share directory. The callback will try and load a local image based on the astronauts name and fallback to using the API's image if one does not exist."
+3. Include the issue number in the description (conveniently, this will automatically resolve the issue upon merging). Describe your motivation, thought process, and solution in the description. For example:
 
-**Finally, don't forget to paste the relevant [Instant Answer Page URL](https://duck.co/ia/new_ia) in the description field**. This will automatically link the PR to the Instant Answer.
+	"**Fixes #2038.** The images used by the API are very large and don't change often. I've but a smaller version of each image (and a 2x version for retina screens) in the share directory. The callback will try and load a local image based on the astronauts name and fallback to using the API's image if one does not exist."
```
Comment (moollaza) [inline CONTRIBUTING.md]: small typo: `I've but a smaller ...`

**Pull request merged**